docs/external-tor-usage.md

```diff
@@ -0,0 +1,49 @@
+# Using External Tor with Haveno
+>Looking to setup external Tor for a seednode? Go [here](https://github.com/haveno-dex/haveno/blob/master/docs/deployment-guide.md#install-tor) for tailored instructions.
+## Setting up Tor:
+
+### 1. Download and Install Tor
+#### Linux
+You can install Tor through your system's package manager, or directly by going to torproject.org. Note that your package manager's version of tor might be outdated or non-GNU (see below note).
+#### MacOS
+You can install Tor through Homebrew (brew install tor), or directly by going to torproject.org. Note that the Homebrew version of tor might be outdated or non-GNU (see note below).
+#### Windows
+You can install Tor directly by going to torproject.org.
+> For best results, ensure that your version of tor matches the one currently used by Haveno. To improve loading times, it is highly recommended to use the GNU build, as it has support for POW. You can check if your build is GNU by running tor --version and looking for a response mentioning GNU.
+
+### 2. Add the following lines to your torrc:
+```
+controlPort=9051
+CookieAuthentication 1
+CookieAuthFile <path to cookie auth file>
+```
+> The "CookieAuthFile" path should be in a relatively stable and safe place. A good place to put it is in the same directory as torrc.
+
+### The location of your torrc file varies depending on your system:
+
+#### Linux
+```/etc/tor/torrc```
+#### MacOS
+```/usr/local/etc/tor/torrc```
+#### Windows
+```\Browser\TorBrowser\Data\Tor\torrc``` (when using the Tor Browser Bundle)
+
+> The above locations are best guesses, and may be incorrect depending on your installation method or platform distribution. In the case that you cannot find the correct directory to place your torrc, tor can be ran in the following format to use a custom torrc: ```tor -f <torrc>```
```

Comment:
Suggested tweaks:

"The above locations may vary depending on your installation method or platform distribution. If you cannot find the correct directory to place your torrc file, you can specify a custom path to torcc in the following format: ```tor -f <path to torcc```